firestore/angular-rewrite/src/environments/environment.ts

```diff
@@ -16,14 +16,10 @@
 
 
 /**  If connecting to a live Firebase project (one that you set up in the 
- * Firebase console) put your config vars into the `prod` field object here
- * and change the `initializeApp` value in `app.module.ts` to `environment.prod`. 
+ * Firebase console) put your config vars into the `environment.prod.ts` file.
  * Otherwise, leave untouched to enable connection to demo project and emulators.
 */
 export const environment = {
```

Comment:
File name or object name? If object name, I may have to rethink the script since I'm relying on having access to the `environment` object but conditionally loading it from either `environment.prod.ts` or `environment.ts`. In the case of file name, I've changed the file name to `environment.emulators.ts` in an upcoming commit!